mu-plugins/global-fonts/index.php

```diff
@@ -86,8 +89,14 @@ function maybe_preload_font( $preload ) {
 		return $preload;
 	}
 
-	foreach ( (array) $style->extra['preload'] as $font_face ) {
-		$font_url = get_font_url( $font_face );
+	$combined_fonts_subsets = array_combine( (array) $style->extra['preload']['fonts'], (array) $style->extra['preload']['subsets'] );
```

Comment:
This is adjusted a bit [here](https://github.com/WordPress/wporg-mu-plugins/pull/298/files#diff-3e55f21b4a7e4e1adfa13b4471290f4cce80b2f75fe9664fa35213977f7fb49cR92-R93) for catering translators' input. The basic idea is the same, more details can be found there.